src/main/kotlin/org/jetbrains/research/tasktracker/tracking/TaskFileHandler.kt

```diff
@@ -101,4 +123,6 @@ object TaskFileHandler {
         }
         return null
     }
+
+    private fun String.pathOrEmpty() = if (!isEmpty()) "/$this" else this
```

Comment:
You can replace `!isEmpty()` with `isNotEmpty()`

Reply:
Also, are you sure about adding `/` ? It can be broken for some OS, I'd prefer to check if the string does not begin from `/` and only in this case add it